packages/engine/src/utils.rs

```diff
@@ -1,23 +1,102 @@
-use std::{env::VarError, time::Duration};
+use std::{env::VarError, fmt::Display, time::Duration};
 
-pub fn init_logger() {
-    use tracing_subscriber::{fmt::time, prelude::*, EnvFilter};
+use log::{Event, Subscriber};
+use tracing_subscriber::{
+    filter::{Directive, LevelFilter},
+    fmt::{
+        self,
+        format::{Format, Writer},
+        time::FormatTime,
+        FmtContext, FormatEvent, FormatFields,
+    },
+    prelude::*,
+    registry::LookupSpan,
+    EnvFilter,
+};
 
-    if cfg!(debug_assertions) && std::env::var("RUST_LOG").is_err() {
-        std::env::set_var(
-            "RUST_LOG",
-            "hash_cloud=debug,hash_engine=debug,cli=debug,server=debug,proto=debug,nano=debug,\
-             apiclient=debug",
-        );
+/// Output format emitted to the terminal
+#[derive(Debug, Copy, Clone, PartialEq, Eq, clap::ArgEnum)]
+pub enum OutputFormat {
+    /// Human-readable, single-line logs for each event that occurs, with the current span context
+    /// displayed before the formatted representation of the event.
+    Full,
+    /// excessively pretty, multi-line logs, optimized for human readability.
+    Pretty,
+    // /// Newline-delimited JSON logs.
+    // Json,
```

Comment:
Can we put a TODO, explaining why its commented out and an asana task to follow up